### DISCUSSION TITLE
profile

### DISCUSSION THREAD

**Discussion by xxxtriple** [community/community#43273] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

como editar legal o profile

**Comment by CodesdaLu** [comment, marked as answer]
na pagina inicial do Github tem uma opcao pronta, mas tem outras formas de deixar ele mais bonito e do seu jeito.

**Reply by xxxtriple** [reply]
ahh obrigada vou procurar no youtube

**Reply by CodesdaLu** [reply]
de nada

**Reply by imran-baitham** [reply]
yeah

**Comment by ettaboyle** [comment]
Hi @xxxtriple & @CodesdaLu, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Thank you!

**Reply by CodesdaLu** [reply]
sorry!

**Reply by xxxtriple** [reply]
@ettaboyle sorry

**Reply by imran-baitham** [reply]
@ettaboyle Nice
